packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -349,6 +349,11 @@ export function TransferPanel() {
       return
     }
 
+    if (!isEOA && !isSmartContractWallet) {
```

Comment:
why not just check if the address is undefined here?

Reply:
I think it should be fine either way, but the `useAccount` hook should cover a bigger scope than just checking for the address, and it also checks for the address itself